pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -59,8 +59,6 @@ type storageClientColumnKey struct {
 	schemaCfg chunk.SchemaConfig
 	client    *bigtable.Client
 	keysFn    keysFn
-
-	distributeKeys bool
```

Comment:
Just noting this is safe to remove because it is never called and the boolean in config is used instead.